torch/nn/modules/container.py

```diff
@@ -419,6 +425,8 @@ def forward(self, x):
                 return x
     """
 
+    _parameters: Dict[str, 'Parameter']  # type: ignore[assignment]
```

Comment:
This should already be on the parents class "Module" no?

Reply:
No, on 'Module',  'Parameter' is optional, while in ParameterList it is mandatory